bundle/manifests/konveyor-operator.clusterserviceversion.yaml

```diff
@@ -20,10 +20,20 @@ metadata:
     categories: Modernization & Migration
     certified: "false"
     containerImage: quay.io/konveyor/tackle2-operator:latest
-    createdAt: "2024-06-03T20:19:41Z"
+    createdAt: "2024-06-06T13:28:50Z"
     description: Konveyor is an open-source application modernization platform that
       helps organizations safely and predictably modernize applications to Kubernetes
       at scale.
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
```

Comment:
Why are we not designed for fips? Maybe we just need to do the exercise to make sure, but I think we should. @dymurray thoughts?

Reply:
@shawn-hurley , I don't know about "designed for", but we've not yet configured downstream to build to run on FIPS enabled clusters.  Until that is done, we should continue to claim "false" for its value.  Franco has a task configure and build MTA (7.0.z & 7.1.z) for it , but it is lower priority than other efforts around MTA at the moment.

Reply:
FWIW, we're already injecting this in downstream CSV.  This PR just brings the change more visibility and the two CSVs closer parity